man/systemd-mount.xml

```diff
@@ -273,6 +281,14 @@
         information.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--tmpfs</option></term>
```

Comment:
(i wouldn't mind if this had a short alias `-T`)

Reply:
added